beets/dbcore/query.py

```diff
@@ -146,10 +146,9 @@ def __init__(self, field, fast=True):
     def col_clause(self):
         return self.field + " IS NULL", ()
 
-    @classmethod
     def match(self, item):
         try:
-            return item[self.field] is None
+            return item[self.field] is None or not item[self.field]
```

Comment:
Well, I did a test and made a lyricsfield='''
So,now it has the field lyrics and it is''
but if I do
```    
 item[self.field] is None
```
it doesn't list it.
but with
```
      not item[self.field]
```
I got them.
So it seems to me that to do what the col_clause  is supposed to do
we should use 
```
      not item[self.field]
```
brrrr....